### PR TITLE
Add skipping files feature when reading directory

### PR DESCRIPTION
**Pull request by neg4n**

### CODE REVIEW

File: src/session.rs [hunk 1]
```diff
@@ -3031,8 +3041,19 @@ impl Session {
...
-                    self.message(format!("Error loading path(s): {}", e), MessageType::Error);
+                }
+
+                match self.edit(paths) {
+                    Ok((success_count, fail_count)) => self.message(
+                        format!(
+                            "Successfully loaded {} path(s), skipped {}",
+                            success_count, fail_count
+                        ),
+                        MessageType::Info,
```
Comment (cloudhead) [inline src/session.rs:3052]: We should only show this message if `success_count + fail_count > 1`.

I'd also prefer something more concise, eg.: "4 path(s) loaded, 5 skipped"

**Comment by cloudhead** [comment]
Thanks for fixing a TODO!

File: src/session.rs [hunk 2]
```diff
@@ -3031,8 +3041,23 @@ impl Session {
             Command::Edit(ref paths) => {
                 if paths.is_empty() {
                     self.unimplemented();
-                } else if let Err(e) = self.edit(paths) {
-                    self.message(format!("Error loading path(s): {}", e), MessageType::Error);
+                }
+
+                match self.edit(paths) {
+                    Ok((success_count, fail_count)) => {
+                        if success_count + fail_count > 0 {
```
Comment (cloudhead) [inline src/session.rs:3048]:
```suggestion
                        if success_count + fail_count > 1 {
```
Otherwise it'll always trigger :)

Comment (neg4n) [reply src/session.rs:3048]: Facts, missclick

**Comment by cloudhead** [comment]
Thanks!

**Pull request merged**